src/content/self-hosted/manage/buildkit-high-performance.mdx

```diff
@@ -68,6 +68,47 @@ buildkit:
     size: 200Gi
 ```
 
+## Dedicated Load Balancer for BuildKit
+
+By default, BuildKit is exposed via gRPC behind the [Okteto Nginx Ingress Controller](self-hosted/helm-configuration.mdx#ingress-nginx).
+This setup simplifies the initial installation, but we recommend exposing BuildKit on a dedicated load balancer for improved performance and control:
+
+**Pros**
+- More control over how build requests are distributed across BuildKit pods.
+- Offloading BuildKit traffic from the Nginx Ingress Controller reduces its network overhead, leading to improved performance for other services relying on the ingress controller
+
+**Cons**
+- There are extra costs associated with provisioning and maintaining an additional load balancer
+
+To expose BuildKit via a dedicated load balancer, add the following to your Okteto Helm configuration:
+
+```yaml
+buildkit:
```

Comment:
I think you also need to specify, If not, I think we have a validation and fail with `"The configuration '.Values.buildkit.ingress.enabled' cannot be set to 'true' if '.Values.buildkit.service.type' is configured as 'LoadBalancer'."`

```
buildkit:
  ingress:
    enabled: false
```

Reply:
updated. we still need to test it in EKS